projects/storefrontapp/src/environments/environment.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference path="../../../../types/typings.d.ts" />
```

Comment:
What is the reason for having to explicitly define the `typings.d.ts`?

Reply:
Moved that reference to tsconfig